.circleci/config.yml

```diff
@@ -537,9 +537,9 @@ jobs:
         environment:
             SOURCE_BRANCH: main
             # TODO: Change to false
-            CREATE_RELEASE_BRANCH: true
+            CREATE_RELEASE_BRANCH: false
```

Comment:
do we need to change this? It shouldn't create a release branch if already exists.